plugin/tslime.vim

```diff
@@ -21,6 +21,9 @@ endfunction
 " Main function.
 " Use it in your script if you want to send text to a tmux session.
 function! Send_to_Tmux(text)
+  if exists("g:tslime_autoset_pane")
```

Comment:
```suggestion
  if exists("g:tslime_autoset_pane") && g:tslime_autoset_pane
```

This allows folks to set `let g:tslime_autoset_pane = 0` to explicitly disable (rather than having to unset).

Reply:
This same change would be used in the other spots too